### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by m-atia**

### CODE REVIEW

File: rules.txt [hunk 1]
```diff
@@ -277,6 +283,7 @@
 /ethicsboard/pages/default.aspx           301  /departments/board-of-ethics/
 /evacuations/(.*)                         301  /services/safety-emergency-preparedness/evacuations/$1
 /executive_orders/?.*                     301  /executiveorders/
```
Comment (karissademi) [inline rules.txt:285]: Can you also update executive_orders to go to the new location? Prevents a double redirect.

**Review by karissademi — CHANGES_REQUESTED** [review]
Minor adjustment

File: rules.txt (hunk 1 above)
Comment (m-atia) [reply rules.txt:285]: done

Comment (karissademi) [reply rules.txt]: Sorry, did you push that change? I'm not seeing it

**Review by karissademi — APPROVED** [review]

**Pull request merged**